pager.c

```diff
@@ -1339,7 +1339,7 @@ static int format_line (struct line_t **lineInfo, int n, unsigned char *buf,
     {
       if (col + 1 > wrap_cols)
 	break;
-      ++col;
+      col += k;
```

Comment:
`k` is used in a wide scope (100 lines) but its never really explained what it does.